packages/logging-bunyan/src/index.js

```diff
@@ -133,6 +134,30 @@ LoggingBunyan.prototype.write = function(record) {
     );
   }
 
+  // Stackdriver Log Viewer picks up the summary line from the 'message' field
+  // of the payload. Unless the user has provided a 'message' property also,
+  // move the 'msg' to 'message'.
+  if (!record.message) {
+    // Clone the object before modifying it.
+    record = extend({}, record);
+
+    // If this is an error, report the full stack trace. This allows Stackdriver
+    // Error Reporting to pick up errors automatically (for severity 'error' or
+    // higher). In this case we leave the 'msg' property intact.
+    // https://cloud.google.com/error-reporting/docs/formatting-error-messages
+    //
+    // TODO(ofrobots): when resource.type is 'global' we need to additionally
+    // provide serviceContext.service as part of the entry for Error Reporting
+    // to automatically pick up the error.
+    if (record.err && record.err.stack) {
+      record.message = record.err.stack;
+    } else if (record.msg) {
```

Comment:
Is it weird to give them two message-like messages as apposed to appending to a single message as in the other case?